<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QLineEdit {
    padding: 5px 10px;
    background-color: #f0f0f0;
    font-size: 10px;
    border: none;
    border-bottom: 1px solid black;
    border-radius: 10px;
    color: black;
}

QPushButton {
    background-color: #012a4a;
    color: white;
    border-radius: 10px;
    min-height: 25px;
    min-width: 15px;
    padding: 5px 7px;
}

QPushButton:hover {
    background-color: #fb8b24;
}

QComboBox {
    border-radius: 5px;
    border: none;
}

#centralwidget {
    background-color: white;
    color: #012a4a;
}

#sec, #msec, #best, #worst {
    font-weight: 800;
    color: #012a4a;
}

#sec, #msec {
    font-size: 120px;
}

#best, #worst {
    font-size: 28px;
}

/* History Table Light Mode */
#history_table {
    background-color: white;
    color: black;
    gridline-color: #012a4a;
    border: 1px solid #012a4a;
}

#history_table QHeaderView::section {
    background-color: #012a4a;
    color: white;
    font-weight: bold;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="maximumSize">
    <size>
     <width>900</width>
     <height>600</height>
    </size>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,0">
    <item>
     <widget class="QWidget" name="nav" native="true">
      <property name="minimumSize">
       <size>
        <width>200</width>
        <height>0</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="button_group" native="true">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>50</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QPushButton" name="mode_btn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../assets/settings-icon-512x512-pk8wypem.png</normaloff>../assets/settings-icon-512x512-pk8wypem.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>29</width>
              <height>35</height>
             </size>
            </property>
            <property name="checkable">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="history_btn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../assets/history.jpg</normaloff>../assets/history.jpg</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>29</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="about_btn">
            <property name="text">
             <string/>
            </property>
            <property name="icon">
             <iconset>
              <normaloff>../assets/about.png</normaloff>../assets/about.png</iconset>
            </property>
            <property name="iconSize">
             <size>
              <width>29</width>
              <height>35</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QTableWidget" name="history_table">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>500</height>
          </size>
         </property>
         <column>
          <property name="text">
           <string>ID</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Time</string>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Type</string>
          </property>
         </column>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="main" native="true">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="widget_5" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Maximum" vsizetype="Maximum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="minimumSize">
          <size>
           <width>650</width>
           <height>0</height>
          </size>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="0,1,0">
          <item>
           <widget class="QComboBox" name="type_cb">
            <property name="minimumSize">
             <size>
              <width>200</width>
              <height>50</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>200</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 20px;
font-weight: 800;
color: white;
border: none;
background-color: #012a4a;</string>
            </property>
            <item>
             <property name="text">
              <string>3x3x3</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>2x2x2</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4x4x4</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5x5x5</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>6x6x6</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>7x7x7</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3x3 bld</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3x3 fm</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3x3 oh</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>clock</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>megaminx</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>pyraminx</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>skewb</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>sq1</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>4x4 bld</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>5x5 bld</string>
             </property>
            </item>
            <item>
             <property name="text">
              <string>3x3 mbld</string>
             </property>
            </item>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="scramble">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">font-size: 12px;
font-weight: 800;
color: white;
border: none;
background-color: #012a4a;</string>
            </property>
            <property name="text">
             <string>R' B' U L2 U' B U' L' F2 U2 L B2 U R2 D' L' B' L2 B' L2</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="scramble_btn">
            <property name="minimumSize">
             <size>
              <width>29</width>
              <height>35</height>
             </size>
            </property>
            <property name="text">
             <string>Scramble</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout">
            <item>
             <spacer name="horizontalSpacer_2">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="sec">
              <property name="minimumSize">
               <size>
                <width>0</width>
                <height>0</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
                <weight>99</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true"/>
              </property>
              <property name="text">
               <string>0</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_2">
              <property name="font">
               <font>
                <pointsize>70</pointsize>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>.</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="msec">
              <property name="font">
               <font>
                <pointsize>-1</pointsize>
                <weight>99</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="text">
               <string>00</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_3">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QLabel" name="best">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>99</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color : white   ;
}</string>
            </property>
            <property name="text">
             <string>Best:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="worst">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>30</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>-1</pointsize>
              <weight>99</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QLabel {
	color : white   ;
}</string>
            </property>
            <property name="text">
             <string>Worst:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QWidget" name="widget_2" native="true">
            <property name="maximumSize">
             <size>
              <width>16777215</width>
              <height>50</height>
             </size>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QPushButton" name="start_btn">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:hover {
	background-color: #fb8b24;
}
QPushButton {
	background-color: #012a4a;
	color : white;
    border-radius: 10px;
	min-height: 25px;
	min-width: 15px;
	padding: 5px 7px;
}
</string>
               </property>
               <property name="text">
                <string>Start</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="stop_btn">
               <property name="font">
                <font>
                 <pointsize>15</pointsize>
                </font>
               </property>
               <property name="styleSheet">
                <string notr="true">QPushButton:hover {
	background-color: #fb8b24;
}
QPushButton {
	background-color: #012a4a;
	color : white;
    border-radius: 10px;
	min-height: 25px;
	min-width: 15px;
	padding: 5px 7px;
}
</string>
               </property>
               <property name="text">
                <string>Stop</string>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
